ex4/VendingDP.Lib/Order.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace VendingDP.Lib
+{
+    public class Order
+    {
+        private readonly List<CombinatedProduct> products;
+        public float Price { get; set; }
+        public string Menu { get; set; }
```

Comment:
The only place you use this variable is in another class `CommandExecutor` for printing the menu. There is no reason to save it here, at least not as `string`

Reply:
After another look you actually need it (not as string). See my discussion at `ItemDetectors` file

Reply:
Resolved